index.html

```diff
@@ -150,5 +150,28 @@ <h2 class="list-of-skills">Skills</h2>
       </div>
     </div>
   </section>
+  <section class="contact-section">
+    <div class="contact-details">
+      <h2 class="paragraph_1">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    </div>
+    <form class="form" action="https://formspree.io/forms" method="post">
+      <ul>
```

Comment:
- [ ] As observed by the previous reviewer, kindly make sure to create an account with [Formspree service](https://formspree.io/html). The {form_id} which is where I underlined in the image below is your unique id which is what formspree would give you when you create the account.

![image](https://user-images.githubusercontent.com/86200356/213118386-9c0ed6e6-e0cf-491d-b0de-47e2c10ff7e6.png)